Softeq.XToolkit.PushNotifications.iOS/Services/IosPushNotificationsConsumer.cs

```diff
@@ -38,6 +39,7 @@ public sealed class IosPushNotificationsConsumer : IIosPushNotificationsConsumer
         /// <param name="showForegroundNotificationsInSystemOptions">Defines how push notification should be presented.</param>
         /// <param name="logManager">Provides logging.</param>
         public IosPushNotificationsConsumer(
+            IPushTokenSynchronizer pushTokenSynchronizer,
```

Comment:
Can you please also update constructor documentation, add new parameter

Reply:
Fixed